lms/envs/devstack.py

```diff
@@ -36,7 +36,6 @@
 ################################ EMAIL ########################################
 
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
-FEATURES['ENABLE_INSTRUCTOR_EMAIL'] = True     # Enable email for all Studio courses
 FEATURES['REQUIRE_COURSE_EMAIL_AUTH'] = False  # Give all courses email (don't require django-admin perms)
```

Comment:
Since we're already in here messing with `bulk_email` settings, should `REQUIRE_COURSE_EMAIL_AUTH` be moved too?